docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -10,9 +10,11 @@ them to run inexpensively in your subscription.
 
 Python support in Azure Functions V2 is in preview and this feature is still immature.
 
-Currently periodic (CRON) and Event Grid functions are supported, but consumption pricing is not
-yet supported.
+- Linux is currently the only supported operating system.
```

Comment:
Also Python 3.6 is a requirement because the linux containers have 3.6 installed.